hbase-replication/src/main/java/org/apache/hadoop/hbase/replication/FSReplicationPeerStorage.java

```diff
@@ -111,6 +114,8 @@ public void addPeer(String peerId, ReplicationPeerConfig peerConfig, boolean ena
       if (!enabled) {
         fs.createNewFile(new Path(peerDir, DISABLED_FILE));
       }
+      write(fs, peerDir, CREATE_TIME_FILE,
```

Comment:
For create time I do not think we need to use the special write method here? As we will never change it...